assets/locales/nl.json

```diff
@@ -1,20 +1,20 @@
 {
   "accessibility": {
-    "irma_logo": "Yivi Logo",
-    "slideshow": "Slide {i} van {n}: {description}",
-    "menu": "Menu",
+    "alert": "Melding",
     "back": "Terug",
     "close": "Sluiten",
+    "collapsed": "Ingeklapt",
+    "expand": "Uitklappen",
+    "expanded": "Uitgevouwen",
+    "irma_logo": "Yivi Logo",
+    "menu": "Menu",
     "next": "Volgende",
-    "alert": "Melding",
     "previous": "Vorige",
-    "expanded": "Uitgevouwen",
-    "collapsed": "Ingeklapt",
-    "unfold": "Uitvouwen",
-    "update": "Bijwerken",
+    "refresh": "Verversen",
     "remove": "Verwijderen",
-    "expand": "Uitklappen",
-    "refresh": "Verversen"
+    "slideshow": "Slide {i} van {n}: {description}",
+    "unfold": "Uitvouwen",
+    "update": "Bijwerken"
   },
```

Comment:
Do you have some auto-organize feature in your IDE by which you changed all these lines? Or do I have to check every change?